src/FeedIo/Adapter/Http/Client.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FeedIo\Adapter\Http;
+
+use DateTime;
+use FeedIo\Adapter\ClientInterface;
+use FeedIo\Adapter\NotFoundException;
+use FeedIo\Adapter\ResponseInterface;
+use FeedIo\Adapter\ServerErrorException;
+use Nyholm\Psr7\Request;
+use Psr\Http\Client\ClientExceptionInterface;
+use Psr\Http\Client\ClientInterface as PsrClientInterface;
+
+class Client implements ClientInterface
```

Comment:
Good point. I removed it, because I think it is better to let the library user configure the client that he/she passes to FeedIO according to the individual needs. The Symfony HTTP Client for example [sets it's own User Agent](https://github.com/symfony/http-client/search?q=user-agent) by default; I guess others might do the same. If you need a specific one to work around some specific limitations, you may as well set it yourself? WDYT?

Reply:
Guzzles also sets a [default user agent](https://github.com/guzzle/guzzle/blob/399c0eaf9bd9b9992bf008b53c758a1f986080b4/src/Client.php#L264)

Reply:
I agree, so let's keep it this way. We just need to make it explicit in the upgrade guide so people won't get surprised by this change

Reply:
I added another line in the migration doc!